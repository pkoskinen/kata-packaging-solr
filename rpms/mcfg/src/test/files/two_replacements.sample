foo = bar
mail = %%EMAIL%%
yes = no
myip = %%MYIP%%
