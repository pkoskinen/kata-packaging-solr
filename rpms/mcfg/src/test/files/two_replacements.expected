foo = bar
mail = postmaster@here.org
yes = no
myip = 1.2.3.4
